jwcrypto/jwa.py

```diff
@@ -693,8 +693,12 @@ def __init__(self):
     def _check_key(self, key):
         if not isinstance(key, JWK):
             raise ValueError('key is not a JWK object')
-        if key.key_type != 'EC':
-            raise InvalidJWEKeyType('EC', key.key_type)
+        if key.key_type not in ['EC', 'OKP']:
+            raise InvalidJWEKeyType('EC or OKP', key.key_type)
+        if key.key_type == 'OKP':
+            if key.key_curve not in ['Ed25519', 'Ed448', 'X25519', 'X448']:
```

Comment:
drop X25519 and X448 for now, I will add them later

Reply:
Done